src/main/java/de/bsi/secvisogram/csaf_cms_backend/json/CommentWrapper.java

```diff
@@ -54,12 +55,45 @@ public static CommentWrapper createNewFromJson(String advisoryId, String newComm
         if (!commentRootNode.has(CommentField.CSAF_NODE_ID.getDbName())) {
             throw new IllegalArgumentException("csafNodeId must be provided!");
         }
+        if (commentRootNode.has(CommentField.ANSWER_TO.getDbName())) {
+            throw new IllegalArgumentException("answerTo must not be provided for comments!");
+        }
         commentRootNode.put(CommentField.ADVISORY_ID.getDbName(), advisoryId);
         commentRootNode.put(CouchDbField.TYPE_FIELD.getDbName(), ObjectType.Comment.name());
 
         return new CommentWrapper(commentRootNode);
     }
 
+    /**
+     * Convert a comment String to an initial CommentWrapper for a given user.
+     * The wrapper has no id and revision.
+     *
+     * @param commentId     the ID of the comment to add the answer to
+     * @param newAnswerJson the new comment in JSON format, requires a commentText field
+     * @return the wrapper
+     */
+    public static CommentWrapper createNewAnswerFromJson(String commentId, String newAnswerJson) throws IOException {
+
+        final ObjectMapper jacksonMapper = new ObjectMapper();
+        final InputStream commentStream = new ByteArrayInputStream(newAnswerJson.getBytes(StandardCharsets.UTF_8));
+        ObjectNode answerRootNode = jacksonMapper.readValue(commentStream, ObjectNode.class);
+
+
+        if (!answerRootNode.has(CommentField.TEXT.getDbName())) {
+            throw new IllegalArgumentException("commentText must be provided!");
+        }
+        if (answerRootNode.has(CommentField.ADVISORY_ID.getDbName())) {
```

Comment:
Wouldn't answers be easier to process if they had an advisoryId? For example when you delete all answers to an andivsory.